style_editor.html

```diff
@@ -8000,6 +8037,13 @@
       <option value=LOCKUP_MELT>Melt</option>
       <option value=LOCKUP_LIGHTNING_BLOCK>LB</option>
     </select>
+    <select name=more id=MORE onchange='OnMoreChange(value)'>
```

Comment:
We really should just add all the effects here.
Maybe add some code to do it automatically instead of listing each one individually...

Reply:
incorporated iterating of the effects the enum